Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
+        {
+            string value = null;
+            if (progInfo != null)
+            {
+                value = GetValue(name + progInfo.Progid.ToString(CultureInfo.InvariantCulture));
```

Comment:
While you were writing this response I was changing what you responded on. :-) What about the following?
> Adding a string-column to settings named something like subsetindicator might be a good solution. It doesn't ruin the concept of a general table for setting, like progid does.